src/components/IncidentList.vue

```diff
@@ -1,42 +1,41 @@
 <template>
-  <center-wrapper>
+  <div>
     <div :class="$style['filter-description']">
-      Showing {{ pluralize(filteredList.length, 'incident', 'incidents') }} in
+      Showing
+      {{ pluralize(filteredList.length, 'incident', 'incidents') }}
+      in
       {{ selectedCityName }}
     </div>
-
     <ul :class="$style.list">
       <incident-list-item
         v-for="incident in filteredList"
         :key="incident.id"
         :incident="incident"
       />
     </ul>
-  </center-wrapper>
+  </div>
 </template>
 
 <script>
 import { computed } from '@vue/composition-api'
 
 import IncidentListItem from '@/components/IncidentListItem.vue'
-import CenterWrapper from '@/components/CenterWrapper.vue'
 import useIncidents from '@/use/incidents'
 
-function pluralize(n, singular, plural) {
-  return `${n} ${n === 1 ? singular : plural}`
-}
-
 export default {
   components: {
     IncidentListItem,
-    CenterWrapper,
   },
 
   props: {
     city: {
       type: String,
       default: '',
     },
+    excludeId: {
+      type: Number,
+      default: 0,
+    },
```

Comment:
Nit: I don't like this `excludeId` very much. It means the parent component knows about the list of incidents too. I prefer that responsibility to lie in a single component, instead of all components being responsible.

To fix this, I'd suggest making `incidents` a prop too.

Reply:
Nice suggestion. Will fix this.

Reply:
This is fixed